test/e2e/systemd_test.go

```diff
@@ -47,11 +51,12 @@ WantedBy=multi-user.target
 
 	})
 
-	It("podman start container by systemd", func() {
-		if os.Getenv("SKIP_USERNS") != "" {
-			Skip("Skip userns tests.")
+	It("podman start container by systemd with selinux enforced", func() {
+		if !selinux.GetEnabled() {
+			Skip("SELinux not enabled")
 		}
-
+		mode := selinux.DefaultEnforceMode()
+		selinux.SetEnforceMode(1)
```

Comment:
could you change it to something like:

```
mode := selinux.EnforceMode()
if mode != selinux.Enforcing {
    err := selinux.SetEnforceMode(selinux.Enforcing)
    Expect(err).To(BeNil())
    defer selinux.SetEnforceMode(mode)
}
```
then you can also drop the `selinux.SetEnforceMode(mode)` at line 92